embeddings/use-cases/outlier-detection/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -60,7 +60,7 @@
    "source": [
     "| | |\n",
     "|-|-|\n",
-    "| Author: | [Roy Arsan](https://github.com/arsan) |"
+    "| Author: | [Roy Arsan](https://linkedin.com/in/arsan) |"
```

Comment:
Can you change this back to GitHub? This could be helpful for automation/auto-assignment of issues in the future.

Reply:
Got it. Updated.